.env.example

```
@@ -33,3 +33,18 @@ NGROK_AUTH_TOKEN=
 
 # External API
 GOOGLE_API_KEY=
+
+# Friendship Score 파라미터 (ANALYSIS-W004). 전부 비우면 정본 기본값 사용
+# 실제 사용값은 분석 결과의 score_params에 그대로 실려 사후 구분이 가능함
+FS_W_SYNC=          # 동조 항 가중치, 기본 1.0
+FS_W_FAA=           # FAA 회피 항 가중치, 기본 0.0 (2단계에서 0.25)
+FS_CORR_METHOD=     # spearman 또는 pearson, 기본 spearman
+FS_SYNC_CHANNELS=   # 동조 대상 채널, 쉼표 구분. 기본 T7,T8,Pz (측두-두정엽)
+                    # none을 주면 채널별 열 대신 CSV 공간평균 열 사용
+FS_SYNC_BAND=       # 동조 대상 대역, 기본 gamma
```

dotenv-linter (4.0.0)

[warning] [SpaceCharacter] The line has spaces around equal sign

(SpaceCharacter)